src/lib/components/RuleForm.svelte

```diff
@@ -20,8 +21,13 @@
 </script>
 
 <div
-	class="flex flex-col flex-grow gap-6 p-4 bg-gradient-to-b from-[#686870] to-black rounded-2xl justify-end"
+	class="relative flex flex-col flex-grow gap-6 p-4 rounded-2xl justify-end overflow-hidden"
+	style="background:linear-gradient(40deg in oklab, black, #002B48 40%, #4FBEFD);"
```

Comment:
optional: the svelte directive is a bit nicer syntax for styles
```suggestion
	style:background="linear-gradient(40deg in oklab, black, #002B48 40%, #4FBEFD);"
```